pgx-utils/src/rewriter.rs

```diff
@@ -505,18 +72,14 @@ impl PgGuardRewriter {
             }
         };
 
-        let sql_graph_entity_submission = entity_submission.cloned().into_iter();
-
         quote_spanned! {func.span()=>
             #prolog
             #[doc(hidden)]
             #vis #sig {
                 #[allow(non_snake_case)]
                 #func
-                #func_call
+                pg_sys::guard::guard( || #func_name(#arg_list) )
```

Comment:
This is effectively the code from line 491, but we're always doing the `no_guard = false` case so I inlined it.